src/index.tsx

```diff
@@ -1,19 +1,17 @@
-import React from 'react';
-import ReactDOM from 'react-dom/client';
+import React from "react"
+import ReactDOM from "react-dom/client"
 import "./styles.scss"
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import App from "./App"
```

Comment:
Try to order your imports.
Import order:
1) Third parties
2) Component
3) Utilities, helper functions
4) Images
5) Styles